Website/DotNetNuke.Website.csproj

```diff
@@ -1114,13 +1114,6 @@
     <Compile Include="Install\Install.aspx.designer.cs">
       <DependentUpon>install.aspx</DependentUpon>
     </Compile>
-    <Compile Include="Install\InstallWizard.aspx.cs">
```

Comment:
Yes, the install wizard was moved outside of the website project, but the file was still listed in the project and was flagged as missing.